x-pack/plugins/security_solution/public/common/containers/source/index.tsx

```diff
@@ -102,7 +102,7 @@ export const useWithSource = (sourceId = 'default', indexToAdd?: string[] | null
     browserFields: EMPTY_BROWSER_FIELDS,
     errorMessage: null,
     indexPattern: getIndexFields(defaultIndex.join(), []),
-    indicesExist: undefined,
+    indicesExist: indicesExistOrDataTemporarilyUnavailable(undefined),
```

Comment:
we need to use this function, or we will always show that you need to set up your data for the app